projects/sbb-esta/angular-business/status/src/status/status.component.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../../angular-core/styles/common/variables';
+@import '../../../../angular-core/styles/common/colors';
+
+:host {
+  display: flex;
```

Comment:
I would expect the status to be an inline element, so in this case flex-inline would be preferred.